titan/params/partnership.yml

```diff
@@ -81,14 +70,6 @@ partnership:
           0:
```

Comment:
nope, this is where I saw it - should this be adherent/non-adherent now instead of bins?